product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/02_query_tool.mdx

```diff
@@ -173,19 +173,19 @@ If the server returns an error, the error message will be displayed on the `Mess
 
 ## Notifications Panel
 
-Use the `Notifications` tab to view the notifications using PostgreSQL `Listen/ Notify` feature. For more details see [PostgreSQL documentation \\&lt;https:// www.postgresql.org/docs/current/sql-listen.html>](<>).
+Use the `Notifications` tab to view the notifications using PostgreSQL `Listen/ Notify` feature. For more details see [PostgreSQL documentation](https:// www.postgresql.org/docs/current/sql-listen.html).
 
 Example:
 
 1.  Execute `LISTEN "foo"` in first `Query Tool` session
 
 <img src="../images/query_output_notifications_listen.png" class="align-center" alt="Query tool notifications listen" />
```

Comment:
The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)